src/ImageSharp/Processing/Processors/Quantization/IFrameQuantizer{TPixel}.cs

```diff
@@ -15,22 +15,18 @@ public interface IFrameQuantizer<TPixel> : IDisposable
         where TPixel : struct, IPixel<TPixel>
     {
         /// <summary>
-        /// Gets a value indicating whether to apply dithering to the output image.
+        /// Gets the quantizer options defining quantization rules.
         /// </summary>
-        bool Dither { get; }
-
-        /// <summary>
-        /// Gets the error diffusion algorithm to apply to the output image.
-        /// </summary>
-        IErrorDiffuser Diffuser { get; }
+        QuantizerOptions Options { get; }
```

Comment:
All usages of `Options` are in the implementing classes, which means that there is probably no point exposing this on the interface.